docs/en/reference/sharding.rst

```diff
@@ -100,11 +100,12 @@ platforms:
 
     <?php
     use Doctrine\DBAL\DriverManager;
+    use Rhumsaa\Uuid\Uuid;
```

Comment:
As you suggest to use `ramsey/uuid` library I guess those namespaces should be: `Ramsey\Uuid\Uuid`

Reply:
Thank you, @stloyd.